### PR TITLE
Update bounce-tracking explainer with feedback and other minor voicin…

### PR DESCRIPTION
**Pull request by wanderview**
…g changes.

### CODE REVIEW

**Comment by wanderview** [comment]
@jyasskin PTAL

File: bounce-tracking-explainer.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ This technique is technically similar to use cases like federated login.  They b
 
 This explainer outlines a proposal for mitigating the privacy impact of bounce trackers.
 
-Safari, Firefox, and Brave have already shipped various bounce tracking mitigations.  These are summarized in a PrivacyCG [draft report](https://privacycg.github.io/nav-tracking-mitigations/#deployed-mitigations).
+Safari, Firefox, and Brave have already shipped various bounce tracking mitigations.  These are summarized in [navigational tracking mitigations draft report](https://privacycg.github.io/nav-tracking-mitigations/#deployed-mitigations).
```
Comment (jyasskin) [inline bounce-tracking-explainer.md:13]:
```suggestion
Safari, Firefox, and Brave have already shipped various bounce tracking mitigations.  These are summarized in the [navigational tracking mitigations draft report](https://privacycg.github.io/nav-tracking-mitigations/#deployed-mitigations).
```

File: bounce-tracking-explainer.md [hunk 2]
```diff
@@ -167,20 +167,24 @@ Some issues that we’d like to discuss:
 
 # Stakeholder Feedback / Opposition
 
-Pending feedback after publication.
+This proposal was discussed at TPAC 2022 in [FedID CG](https://github.com/fedidcg/meetings/blob/main/2022/2022-09-12-TPAC-notes.md#bounce-tracking-mitigations), [Web Payments WG](https://www.w3.org/2022/09/13-wpwg-minutes.html#:~:text=running%20their%20businesses.-,Navigation%20tracking,-%5BBen%27s%20slides), and in informal settings.
```
Comment (jyasskin) [inline bounce-tracking-explainer.md:170]: There's an id for this section:
```suggestion
This proposal was discussed at TPAC 2022 in [FedID CG](https://github.com/fedidcg/meetings/blob/main/2022/2022-09-12-TPAC-notes.md#bounce-tracking-mitigations), [Web Payments WG](https://www.w3.org/2022/09/13-wpwg-minutes.html#t02), and in informal settings.
```

**Review by jyasskin — APPROVED** [review]
LGTM. @pes10k, how are you feeling about the FPS section sticking around?

**Comment by pes10k** [comment]
im okay with leaving the FPS text in

**Pull request merged**